ibm/service/power/resource_ibm_pi_instance.go

```diff
@@ -560,6 +560,8 @@ func resourceIBMPIInstanceRead(ctx context.Context, d *schema.ResourceData, meta
 	}
 	if powervmdata.Fault != nil {
 		d.Set(Attr_Fault, flattenPvmInstanceFault(powervmdata.Fault))
+	} else {
+		d.Set(Attr_Fault, nil)
```

Comment:
Did n't get it why we need to initialize to nil

Reply:
If it doesn't get initialized, the terraform apply keeps thinking there's a change to apply, and tries to apply it again, but doesn't do anything. @surajibm found the issue